substrate/frame/Cargo.toml

```diff
@@ -0,0 +1,91 @@
+[package]
+name = "frame"
+version = "0.0.1-dev"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+license = "Apache-2.0"
+homepage = "paritytech.github.io"
+repository.workspace = true
+description = "The single package to get you started with building frame pallets and runtimes"
+
+[package.metadata.docs.rs]
+# enable `experimental` feature for docs
+features = ["experimental"]
```

Comment:
Wondering why we're releasing this in 'experimental', and if there is a quantifiable checklist to complete to get it out of 'experimental' mode.

Reply:
The reason behind publishing this in the current state is to start referencing to it on documentation, examples, etc...
@kianenigma do you have this checklist in mind?

Reply:
Is documentation being built with the `--experiemental` flag?

Reply:
yes it is https://github.com/paritytech/polkadot-sdk/pull/1337#discussion_r1363702133